ecs.tf

```diff
@@ -78,6 +78,44 @@ module "app_container_definition" {
   }
 }
 
+module "datadog_container_definition" {
+  source                   = "git::https://github.com/cloudposse/terraform-aws-ecs-container-definition.git?ref=tags/0.21.0"
+  container_name           = "datadog-agent"
+  container_image          = "datadog/agent:latest"
```

Comment:
I think we should fix the agent version instead of using latest.

Reply:
a solid point - I'll update now